cluster-provision/gocli/Makefile

```diff
@@ -1,20 +1,20 @@
 SHELL := /bin/bash
 
-BIN_DIR = $(CURDIR)/build
-GO ?= go
+IMAGES_FILE ?= images.json
 
 export GO111MODULE=on
 export GOPROXY=direct
 export GOSUMDB=off
 export GOFLAGS=-mod=vendor
 
+BIN_DIR = $(CURDIR)/build
+GO ?= go
 
 all: container-run
 
 .PHONY: gocli
-cli: fmt
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 $(GO) build -o $(BIN_DIR)/cli ./cmd/cli
-
+cli:
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 $(GO) build -ldflags "-X 'kubevirt.io/kubevirtci/cluster-provision/gocli/images.SUFFIXES=$(shell cat $(IMAGES_FILE)|base64 -w 0)'" -o $(BIN_DIR)/cli ./cmd/cli
```

Comment:
Why you prefer to use `ldflags` instead of hard-coding this in as constants or something in a dedicated constants module?

Reply:
The idea is to not store hashes at repo at all when release pipeline is implemented, the release pipeline will just bump whatever digest is need to be bump and pass a json file to this.